framework/include/materials/DerivativeFunctionMaterialBase.h

```diff
@@ -27,8 +27,8 @@ InputParameters validParams<DerivativeFunctionMaterialBase>();
  *
  * A DerivativeFunctionMaterialBase provides numerous material properties which contain
  * the free energy and its derivatives. The material property names are
```

Comment:
since this is in the framework now, should this say free energy?

Reply:
fair point!